mixer/adapter/list/config/config.proto

```diff
@@ -58,6 +58,9 @@ message Params {
 
         // List entries are treated as IP addresses and ranges.
         IP_ADDRESSES = 2;
+
+        // List entries are treated as regex.
+        REGEX = 3;
```

Comment:
What kind of regular expression do we support here? Can you please give a few examples.

Reply:
Comment added.